include/kokkos-utils/callbacks/Events.hpp

```diff
@@ -303,10 +303,14 @@ using paired_event_t = typename PairedEventType<EventType>::type;
 
 //! @name Stream operators.
 ///@{
-//! Helper function related to the implementation of the stream operators.
-template <Event EventType>
+/**
+ * @brief Helper function related to the implementation of the stream operators that removes the @c Kokkos::utils::callbacks:: prefix.
```

Comment:
```suggestion
 * @brief Helper function to remove the @c Kokkos::utils::callbacks:: prefix.
```
(best to keep the `that` right after the noun if you keep the subsentence with "that ...")

Reply:
Done.